### PR TITLE
Edit check property feature for new check flag and handle extra parameters

### PR DESCRIPTION
**Pull request by ngdeqi**

### CODE REVIEW

File: src/main/java/seedu/duke/Messages.java [hunk 1]
```diff
@@ -50,6 +50,16 @@ public class Messages {
...
             + "the following client(s) will be deleted:";
 
+    public static final String MESSAGE_CHECK_PROPERTY = "Showing check results for this property:";
+
+    public static final String MESSAGE_CHECK_PROPERTY_RESULT = "Here are the tenants renting this property:";
+
+
+    public static final String MESSAGE_CHECK_CLIENT = "Showing check results for this client:";
+
+    public static final String MESSAGE_CHECK_CLIENT_RESULT = "Here is the property this client is renting:";
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/Messages.java]: Good job with the clear and concise message to the user.

File: src/main/java/seedu/duke/exception/check/CheckExtraParametersException.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+    public CheckExtraParametersException(String extraArgument) {
+        this.extraArgument = extraArgument;
+    }
+
+    @Override
+    public String toString() {
+        return MESSAGE_CHECK_EXTRA_ARGUMENTS + extraArgument + System.lineSeparator()
+                + MESSAGE_CHECK_PROPERTY_WRONG_FORMAT
+                + MESSAGE_CHECK_CLIENT_WRONG_FORMAT;
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/exception/check/CheckExtraParametersException.java:23]: Good use of different message to integrate together. :D

**Review by wilsonngja — COMMENTED** [review]
Overall, the tags that's requires changes seems like has been done thoroughly without missing out any. Great job and LGTM :D

**Review by FeliciaBeatrice — APPROVED** [review]
LGTM!

**Pull request merged**